src/Conversion/ONNXToKrnl/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ add_library(OMONNXToKrnl
         Tensor/Squeeze.cpp
         Tensor/Unsqueeze.cpp
         Tensor/Constant.cpp
+        Tensor/ConstantOfShape.cpp
         Tensor/Concat.cpp
         Tensor/Split.cpp
         ConvertONNXToKrnl.cpp)
```

Comment:
Nit. These should be placed in sorted order for easier navigation.

Reply:
Could you explain what order did you mean? Is it alphabetical order?

Reply:
Yes.

Reply:
I am sorry I don't understand what is your suggestion specifically? I put ConstanOfShape after Constant to follow the alphabetical order. I am aware of that the whole list currently does not follow the order, and we will fix this later.

Reply:
@tungld Yes, the above change that you have done was my suggestion. Thanks.